Jenkinsfile

```diff
@@ -1,3 +1,8 @@
+String tfProject      = "nia"
+String tfEnvironment  = "build1"
+String tfComponent    = "gpc-consumer"
+Boolean publishWiremockImage = false // true: To publish gpc-consumer wiremock image to AWS ECR gp2gp-wiremock
```

Comment:
on what conditions we should build and publish gpc wiremock?

Reply:
this has not been clarified yet but I found info about wiremock in Environment variables section (https://gpitbjss.atlassian.net/browse/NIAD-1154) . So I have added up a control to build wiremock image.